rpcd/playbooks/roles/elasticsearch/files/reindex-liberty.py

```diff
@@ -0,0 +1,306 @@
+            else:
+                print("Skipping: {} reindexing in progress").format(index)
+
+    else:
+        reindex_params = 'http://' + es_host + '/' + parsed_args.index + \
+            '/_reindex/' + parsed_args.index + parsed_args.suffix
+        print("Reindexing Single Index: {0} into: {1}").format(
+            parsed_args.index,
+            parsed_args.index + parsed_args.suffix)
+        if not parsed_args.dry_run:
+            requests.post(reindex_params)
+
+
+def clean_legacy(es, parsed_args):
+    """Drops the -liberty indices created by a previous reindex."""
+    if parsed_args.dry_run:
+        print("Dry run, no operations will be performed")
+    for index in get_indices(es, parsed_args, True):
+        if parsed_args.suffix in index:
+            if not parsed_args.dry_run:
+                es.indices.delete(index)
+            print ("Deleted: {}").format(index)
+
+
+def monitor_reindex(es, parsed_args):
+    """Monitor the reindexing process."""
+    total_count = 0
+    done_count = 0
+    stats = get_stats(es)
+    indices = get_indices(es, parsed_args, True)
+    counts = None
+    counts = {}
+    for index in indices:
+        if parsed_args.suffix in index:
+            master_name = index.replace(parsed_args.suffix, "")
+            counts[master_name] = _calc_pct(es,
+                                            parsed_args,
+                                            stats,
+                                            master_name,
+                                            index)
+
+    for master in counts.keys():
+        slave_index = master + parsed_args.suffix
+        master_count = counts[master][0]
+        slave_count = counts[master][1]
+        reindex_pct = counts[master][2]
+        if parsed_args.verbose:
+            print("Master Index: {0:15s} "
+                  "Slave Index: {1:15s} "
+                  "Master Count: {2:10d} "
+                  "Slave Count: {3:10d} "
+                  "Percentage Complete: {4:3.2f}%").format(master,
+                                                           slave_index,
+                                                           master_count,
+                                                           slave_count,
+                                                           reindex_pct)
+        total_count += master_count
+        done_count += slave_count
+    try:
+        pct_done = (float(done_count) / float(total_count)) * 100
+    except ZeroDivisionError:
+        pct_done = float(0.00)
+    if parsed_args.verbose:
+        print("Total Docments: {0:20d} "
+              "Reindexed Documents: {1:20d} "
+              "Percentage complete: {2:3.2f}%").format(total_count,
+                                                       done_count,
+                                                       pct_done)
+    else:
+        print("Percent complete: {0:3.2f}%").format(pct_done)
+    if pct_done == 100:
+        exit(code=0)
```

Comment:
According to https://docs.python.org/2.7/library/constants.html#exit the site builtin `exit` shouldn't be used outside of the interactive interpreter shell. Consider using `sys.exit` instead. Also, `code` is the implicit first parameter and doesn't have to be specified if you choose to continue using `exit`.